### PR TITLE
Generate public host verification links

### PR DESCRIPTION
**Pull request by gewfy**
I haven't tested this locally. But tests and types are saying it should work :D
image

### CODE REVIEW

File: functions/src/lib/slack.spec.ts [hunk 1]
```diff
@@ -107,7 +108,7 @@ describe('slack', () => {
             type: 'section',
             text: {
               type: 'mrkdwn',
-              text: '*Accepted <mailto:some@email.com?body=123456|some@email.com> as public host, please send code `123456` to the user.*',
+              text: '*Accepted <mailto:some@email.com?body=123456|some@email.com> as public host, please send code `123456` http://some.deep/verification/link to the user.*',
```
Comment (swemail) [inline functions/src/lib/slack.spec.ts:111]: Maybe add it also t the mailto body?

Comment (swemail) [reply functions/src/lib/slack.spec.ts:111]: Sorry, my bad, just a test

**Review by swemail — APPROVED** [review]
Nice addition this!

File: functions/src/lib/slack.spec.ts (hunk 1 above)
Comment (gewfy) [reply functions/src/lib/slack.spec.ts:111]: Ah! Will fix :)

**Pull request merged**